lib/account/account-manager.js

```diff
@@ -19,6 +36,28 @@
  * @typedef {Object} accountManager~confirmedInfo
  * @property {string} mfaKey
  * @property {string} passwordSalt
+ * @property {accountManager~hashSet} passwordHashing
```

Comment:
I don't see this property used anywhere.

The name of this is "confirmedInfo".  They should be delivered the password hashing information before the registration is turned into a full account.  Confirmed = email confirmed and they are finishing signup, yes?